projects/pillow/build_depends.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2020 Eric Soroos
```

Comment:
This need to be copyright "Google LLC"
(if you don't want to do this, I suggest putting the file outside of the OSS-Fuzz repo and pulling it in)

Reply:
nit: also 2021

Reply:
I don't really care, it's not that big of a script, but it might wind up elsewhere. 

However, 

* The tests asks for a license block, they do not state what license. 
* The contributing.md doesn't mention this, and specifically says: 
  > The CLA is necessary mainly because you own the copyright to your changes,

Reply:
Can you use this license header. We can improve CONTRIBUTING.md later to mention this, any patches welcome to add it to that documentation

```
# Copyright 2021 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing perepo_managerissions and
# limitations under the License.
```

Reply:
`perepo_managerissions`?

IANAL, but ... That doesn't look right. 

I'm not even sure what autocorrect was on when it did that.

Reply:
Here is corrected one

```
# Copyright 2021 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```